### DISCUSSION TITLE
Is this working with react-native 0.66?

### DISCUSSION THREAD

**Discussion by zoolle** [software-mansion/react-native-reanimated#2535] [Q&A]
I am trying to use the latest version `react-native-reanimated@2.3.0-alpha.3` with `react-native 0.66` but when I run `yarn android` I get this:

`Failed to transform react-native-reanimated-65-jsc.aar (project :react-native-reanimated) to match attributes {artifactType=android-aar-metadata}.`

**Comment by DanielMarkiel** [comment]
@zoolle `react-native-reanimated@2.3.0-alpha.3` is not the latest version. Please check the releases section before asking -> https://github.com/software-mansion/react-native-reanimated/releases
Btw. The latest version is https://github.com/software-mansion/react-native-reanimated/releases/tag/2.3.0-beta.2 (check the release notes)
